styles/_govuk-elements.scss

```diff
@@ -0,0 +1,30 @@
+// GOV.UK elements
+@import "govuk-elements-sass/public/sass/elements/helpers";                       // Helper functions and classes
```

Comment:
It didn't work before because I forgot the `;` when I did the import.  Now it does, cheers